### ISSUE TITLE
Billing RDS

### ISSUE THREAD

**Issue by pierreozoux** [operator-framework/operator-metering#554]
I'm a multi-tenant openshift admin.
I'm working on figuring out the best way to provide RDS as a service for our users, and [it will probably be with some kind of form of operators](https://github.com/awslabs/aws-service-operator/issues/137#issuecomment-451868081).

Then, the next natural question is how do I charge back my customers.

This is how I stumbled upon this project.

We already developed an internal tool for charge back. It is based on prometheus data.
Basically if I had an exporter that would tell me how many CRDs are there, on which namespace, and some details about it, I should be fine.

I'm investigating if the operator metering could help me. And is it made for my usage?

Thanks a lot for you insight!

**Comment by chancez** [comment]
That would definitely work well with the way metering currently works! You would end up creating a ReportPrometheusQuery CR which defines the Prometheus query to run periodically to import data, then you can define ReportGenerationQueries and Reports to further post-process it using SQL.
